deployment/deploy.yml

```diff
@@ -190,3 +192,6 @@ parameters:
   name: RUN_BUNDLE_SYNC
   required: false
   value: 'false'
+- description: Application allow list for new bundles with skus to update the feature in the IT features API
+  name: BUNDLES_SKUS_ALLOW_LIST
+  value: ansible, smart_management_rods, rhoam, rhosak, openshift, rhel
```

Comment:
I think we just need to remove the spaces there - from what it was set to before I imagine it's splitting on the `","` character.